packages/fxa-payments-server/src/components/NewUserEmailForm/en.ftl

```diff
@@ -8,7 +8,10 @@ new-user-enter-email =
   .label = Enter your email
 new-user-confirm-email =
   .label = Confirm your email
-new-user-subscribe-product-updates = I’d like to receive product updates from { -brand-name-firefox }
+new-user-subscribe-product-updates-mozilla = I’d like to receive product news and updates from { -brand-mozilla }
+new-user-subscribe-product-updates-snp = I’d like to receive security & privacy news and updates from { -brand-mozilla }
```

Comment:
Not blocking for localization, but I think the guideline is to avoid ampersands
https://acorn.firefox.com/latest/content/punctuation.html

Reply:
Ah, thank you so much. Checked in with Product and made the change to remove the `&`.